From 07a0a423a8469a2dd6d7f64bb3b62ba6ac42cc28 Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Fri, 20 Jul 2018 15:20:11 +0200
Subject: [PATCH] GooString needs to be const since >=poppler-0.64.0

This only fails with >=poppler-0.67.0 but the change to const was done
in 0.64.0
---
 filter/pdf.cxx | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/filter/pdf.cxx b/filter/pdf.cxx
index 206ccf88..665eab09 100644
--- filter/pdf.cxx
+++ filter/pdf.cxx
@@ -734,6 +734,9 @@ extern "C" int pdf_fill_form(pdf_t *doc, opt_t *opt)
         }
 
         FormField *ff = fm_text->getField();
+#if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 64
+        const
+#endif
         GooString *field_name;
         field_name = ff->getFullyQualifiedName();
         if ( ! field_name )

